x-pack/plugins/fleet/server/services/preconfiguration.test.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import uuid from 'uuid';
+
```

Comment:
This diff is best viewed with GitHub's "hide whitespace" option enabled, as we introduced a new `describe` block here and shifted a lot of large test blocks over.